packages/plugin-list/test/Checklist.test.ts

```diff
@@ -2386,8 +2425,13 @@ describePlugin(List, testEditor => {
                             contentBefore:
                                 '<ul class="checklist"><li class="checked">ab[cd</li><li style="list-style: none;"><ul class="checklist"><li class="unchecked">ef]gh</li></ul></li></ul>',
                             stepFunction: deleteBackward,
+                            // The indented list cannot be unchecked while its
+                            // parent is checked: it gets checked automatically
+                            // as a result. So "efgh" gets rendered as checked.
+                            // Given that the parent list item was explicitely
+                            // set as "checked", that status is preserved.
```

Comment:
Well this test was wrong to begin with since the initial situation does not make sense. Nothing to do here, it's a good test to check our "resolution" of this issue, but the original test by chm was wrong IMO.